infobip-typescript-generator-extension-api/src/test/java/com/infobip/typescript/validation/ClassValidatorDecoratorTestBase.java

```diff
@@ -9,10 +9,13 @@
 
 abstract class ClassValidatorDecoratorTestBase extends TestBase {
 
+    private static final String IMPORT_COMMON_VALIDATION_MESSAGES = "import { CommonValidationMessages } from 'infobip-typescript-generator-common'";
+    private static final String IMPORT_CLASS_VALIDATOR = "import { ValidateNested, IsDefined, IsOptional, IsNotEmpty, MaxLength, MinLength, Max, Min, ArrayMaxSize, ArrayMinSize } from 'class-validator'";
+    protected static final String IMPORTS = IMPORT_COMMON_VALIDATION_MESSAGES + ";\n" + IMPORT_CLASS_VALIDATOR + ";\n";
```

Comment:
Those 3 variables make tests really hard to read. I'd rather have duplication with better readability than having to jump all over the place to figure out what the expected value is.

Reply:
You think so? My reasoning with this change was that 
1) when there's less code it's more readable. 
2) these imports were already getting passed to the `TestBase` constructor inside `ClassValidatorDecoratorTestBase` so the test does not test these values, they're just part of test framework setup. If they're not part of the test they should not be visible in the test and the test author should ideally not think about it.

Let me know if this argumentation convinced you, I'll bring back the old version if not.

Reply:
Less code doesn't automatically mean it's more readable - that's a false assumption. There's no strict correlation.
I'm not sure I follow you on 2. but my point is on removed lines in then statements. Now it's a concatenation of those fields. Combination of those concatenation and 3 separate fields makes it really hard to follow what is going on.

Reply:
I've just remembered why I changed it: it's because I added `"@IsOptional("` to `ClassValidatorDecoratorExtension.DEFAULT_VALIDATIONS`, reran the test and the test still passed, even though `@IsOptional` was not in its imports.

Reply:
So then I had to figure out what was going on and I discovered that the imports were actually getting taken from `ClassValidatorDecoratorTestBase`

Reply:
Let me share my point of view, maybe it will help maybe it will not :)
1. I share my point of view here with @lpandzic. To be honest I have nothing to add then just to agree on fact that less code == more readable code is false assumption 

2. @PiotrMachner i think that I am following you, you have extract this part of code mainly for reusability, because you need to configure ClassValidatorDecoratorTestBase witch takes instance of ClassValidatorDecoratorExtension + import fields (as you said this is configuration of test base).
I am following your logic, but here is one but, this project (at least from my experience) is very complex and from time to time it is hard to follow some parts.

Based on that I would also sided with @lpandzic on comment, mainly because when someone else is going to work with this code test should be clear as much as possible (and here by clear I mean, not distracting) and scrolling or jumping in this case is distracting and one can lose focus while being in debug session of already complex code :)

TL;DR; I would revert to fully stating imports, just to make test easy to follow :)

Reply:
Also i have seen now that @PiotrMachner you replied like hour a go but for some reason i did not see comment. I hate git user experience on reviewing code :)
But for problem mentioned in your above comment ->
How it is solved now ? As I can see solution were to detect where test is located and then to configure list of imports in test (or am i missing something) ?

Why am i asking that, because if we need to still locate tests, when we add something in code and manually add changes, we are still at square one, test could pass even if we have change something, we have just "grouped" imports in static variables, meaning that beside needing to jump from test to static to global variables we did not get any benefits ? (or am i missing something)

Reply:
@lpandzic @msertic1 I've brought back the old version (with the full imports), adding only the new import for `IsOptional` 👍